Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -1641,6 +1656,60 @@ def checkpoint_show_task_command(client: Client, task_id: str) -> CommandResults
     return command_results
 
 
+def checkpoint_add_objects_batch_command(client: Client, object_type: str, ipaddress, name):
```

Comment:
If you add new commands you need to update the readme with those new commands
See https://xsoar.pan.dev/docs/documentation/readme_file/#documentation-must-be-generated-if